packages/providers/src/operations.graphql

```diff
@@ -55,7 +55,7 @@ fragment coinFragment on Coin {
   amount
   assetId
   maturity
-  status
+  coinStatus: status
```

Comment:
Why we had to add this alias here?

Reply:
because both coins and messages get combined into a resource object, but unlike values like `amount`, `status` is different between the two (one is type `CoinStatus` and the other `MessageStatus`.